upload_package

```diff
@@ -85,6 +95,15 @@ fi
 
 echo "Uploading update payload"
 
+EMBED_EXTRA=()
+for EXTRA_FILE in "${EXTRA_FILES[@]}"; do
+  E_NAME=$(basename "${EXTRA_FILE}")
+  E_SIZE=$(stat --format='%s' "${EXTRA_FILE}")
+  E_HASH_SHA1=$(openssl dgst -sha1 -binary < "${EXTRA_FILE}" | base64)
```

Comment:
I suppose that sha1sum can't be used due to some update_engine/omaha/nebraska weirdness?

Reply:
Yes, it has no "--base64" flag :D